learn/getting-started/working-with-data-in-ballerina.md

```diff
@@ -10,25 +10,19 @@ redirect_from:
   - /learn/working-with-data-in-ballerina
 ---
 
-Ballerina supports first-class support for writing SQL-like queries to process data.
-
-Language-integrated queries can process any Ballerina iterable.
+Ballerina supports first-class support for writing SQL-like queries to process data. Language-integrated queries can process any Ballerina iterable.
 
 In this tutorial, you will be writing queries to filter, sort, and join with different data sets and produce new data sets.
 
-1. Creating a new Ballerina project.
-2. Defining the Covid dataset to be processed.
-3. Filtering all countries which have more than 10,000,000 Covid cases.
-4. Sorting the top three countries by the number of reported Covid cases.
-5. Joining the table with an array to find the number of recovered patients.
-6. Finding any discrepancies in reported Covid dataset while maintaining intermediate states.
-
-## Prerequisite
+## Setting up the Prerequisite
 
-Following two prerequisites are needed for this tutorial. 
+To complete this tutorial, you need the below.
```

Comment:
Here we have prerequisites, setting up the prerequisites, and to complete this tutorial, you need the below. Prerequisites alone would suffice.

Reply:
We need to have the `ing` form in the topics and that is why we say `Setting up the Prerequisites` to match with the other titles. 

We can't omit the `to complete this tutorial, you need the below` section as just having the list below the topic would not give the complete meaning that the user needs to set them up as per the topic.

Reply:
Let's revise this format in the future then. +1 to go ahead.

Reply:
Fixed as per https://github.com/ballerina-platform/ballerina-dev-website/pull/3564#discussion_r794990859.